pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -376,7 +386,29 @@ func assertPayloadsBaseErr(
 
 	var actualFormatted []string
 	for _, m := range actual {
-		actualFormatted = append(actualFormatted, m.String())
+		if useProtobuf {
+			var msg changefeedpb.Message
+			if err := protoutil.Unmarshal(m.Value, &msg); err != nil {
+				return err
+			}
+			valJSON, err := gojson.Marshal(msg.GetBare())
+			if err != nil {
+				return err
+			}
+
+			var key changefeedpb.Key
+			if err := protoutil.Unmarshal(m.Key, &key); err != nil {
+				return err
+			}
+			keyJSON, err := gojson.Marshal(key.Key)
+			if err != nil {
+				return err
+			}
+
+			actualFormatted = append(actualFormatted, fmt.Sprintf(`%s: %s->%s`, m.Topic, keyJSON, valJSON))
```

Comment:
can you instead replace `m.Key`, `m.Value`, and still have the `m.String()` call happen? this preserves some other stuff in there